src/lib/Libwin/passwd.c

```diff
@@ -5411,6 +5152,11 @@ has_read_access_domain_users(wchar_t dctrlw[PBS_MAXHOSTNAME+1])
 			(netst == ERROR_LOGON_FAILURE)) {
 			log_eventf(PBSEVENT_ERROR, PBS_EVENTCLASS_SERVER, LOG_ERR, __func__, "failed in NetUserGetInfo, with errno %d", netst);
 			goto has_read_access_domain_users_end;
+		} else {
+			if ((netst == ERROR_BAD_NETPATH) || (netst == ERROR_INVALID_LEVEL) ||
```

Comment:
since u are logging at DEBUG3 you can actually log all error cases - not just the specific 4 u are checking for. BTW, only one code is left it seems ERROR_ACCESS_DENIED :)

Reply:
As we are printing this in a loop, so if we want to log for all the error codes, I think it will be better to increase the debug level to DEBUG4

Reply:
DEBUG3 would have been good enough, but its ok at DEBUG4 also